backend/main.go

```diff
@@ -280,23 +353,30 @@ func (s *Server) handlePutPeriod(w http.ResponseWriter, r *http.Request) {
 	if !ok {
 		return
 	}
-	if !s.ensureTeamExistence(w, r, teamID, true) {
+	team, exists := s.ensureTeamExistence(w, r, teamID, true)
+	if !exists {
 		return
 	}
+	log.Print(team)
```

Comment:
Why are we logging this?

Reply:
😬 oops, this was definitely for debugging.